### PR TITLE
Create Non-EU residents

### PR DESCRIPTION
**Pull request by estherrgarcia**

### CODE REVIEW

File: talent/onboarding/README.md [hunk 1]
```diff
@@ -89,4 +89,4 @@ At the San Francisco office, we provide:
 
 #### San Francisco Conference Rooms
 
-- Since we're sharing the space with Holberton, the rooms are on use them as you need basis (they can't be reserved).
\ No newline at end of file
+- Since we're sharing the space with Holberton, the rooms are on use them as you need basis (they can't be reserved).
```
Comment (tsolakoua) [inline talent/onboarding/README.md]: @estherrgarcia  The part of "the rooms are on use them as you need basis" is not very clear to me

Comment (eiso) [reply talent/onboarding/README.md]: @tsolakoua it means, that we can make use of their conference rooms but we don't have a dedicated source{d} conf. room there yet.

Comment (tsolakoua) [reply talent/onboarding/README.md]: @eiso thanks! it seems that there is something wrong with the syntax of the sentence and doesn't make it very clear, at least to me. Not sure if needs to be changed.

Comment (eiso) [reply talent/onboarding/README.md]: Please feel free to change it ;)

**Pull request merged**